src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -124,6 +134,19 @@ const TeamMembers = createReactClass({
     );
   },
 
+  removeButton(member) {
+    return (
+      <Button size="small" onClick={this.removeMember.bind(this, member)}>
+        <InlineSvg
+          src="icon-circle-subtract"
+          size="1.25em"
+          style={{marginRight: '0.5em'}}
+        />
+        {t('Remove')}
```

Comment:
Love the new icon, wish we had a better word for "remove" since it's more like "disassociate" rather than "destroy" (which subtract helps convey a lot better than the trash bin we use elsewhere)